protostar/cli/output_command_util.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+from .protostar_argument import ProtostarArgument
+from ..io import Messenger, JsonMessenger, HumanMessenger, LogColorProvider
+
+
+class OutputCommandUtil:
+    output_arguments = [
```

Comment:
[nitpick]: this should be uppercase since it's a constant

Reply:
done